plugin/src/test/resources/widget/slider/_cq_dialog.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+          xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+          jcr:primaryType="nt:unstructured"
+          sling:resourceType="cq/gui/components/authoring/dialog"
+          jcr:title="Slider Widget Dialog"
+          width="{Double}560.0"
+          height="{Double}480.0">
+    <content jcr:primaryType="nt:unstructured" sling:resourceType="granite/ui/components/coral/foundation/container">
+        <layout jcr:primaryType="nt:unstructured"
+                sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"/>
+        <items jcr:primaryType="nt:unstructured">
+            <column jcr:primaryType="nt:unstructured"
+                    sling:resourceType="granite/ui/components/coral/foundation/container">
+                <items jcr:primaryType="nt:unstructured">
+                    <number jcr:primaryType="nt:unstructured"
+                            sling:resourceType="etoolbox-authoring-kit/components/authoring/slider"
+                            name="./number"
+                            fieldLabel="Number"
+                            renderHidden="{Boolean}false"
+                            required="{Boolean}false"
+                            min="{Long}10"
+                            max="{Long}200"
+                            step="{Long}2"
+                            orientation="vertical"
+                            filled="{Boolean}true"
```

Comment:
oriental should be VERTICAL like in plugin/src/test/com/exadel/aem/toolkit/test/widget/SliderWidget.java and add disabled="{Boolean}false"